src/dataTypes/mosString128.ts

```diff
@@ -8,17 +7,18 @@ export class MosString128 {
 	}
 	/** */
 	toString (): string {
-		return this._str
+		return this._str !== `undefined` ? this._str : ''
```

Comment:
This seems very icky. I think it would be better to check for undefined values in the setter at line 23/24, so that `undefined` values aren't stringified into `'undefined'`

Reply:
Sure, you are right i put this code to when you are debugging but forgot the move to the appropriate location.